runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/io/UnboundedSourceWrapper.java

```diff
@@ -158,7 +159,7 @@ public void run(SourceContext<WindowedValue<OutputT>> ctx) throws Exception {
     }
 
     LOG.info("Unbounded Flink Source {}/{} is reading from sources: {}",
-        subtaskIndex,
+        subtaskIndex + 1,
```

Comment:
Why did you add this? I know it's very confusing that "0/1" means first source of one but that's how it is in all of Flink with the subtask index. Maybe we should keep that for consistency.